cs481-ekh/f21-welolo
# Updated all cypress tests to pass after changes to code #74

Verification Steps:

1. Run both client and server using "npm start" command
2. Run Cypress using "npx cypress open" inside Welolo folder
3. Ensure all the tests pass
